src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -1786,26 +1788,28 @@ bool processOnChangeCondition
         if (condValues != NULL) {
             /* Check if some of the attributes in the NotifyCondition values list are in the entity.
              * Note that in this case we do a query for all the attributes, not restricted to attrV */
-            ContextElementResponseVector rawCerV;
+            ContextElementResponseVector rawCerV2;
```

Comment:
No, rawCerV cannot be used.
That name is already used in this function, a few lines before.
If you dont like rawCerV2, let's invent some other variable name ...